app/_data/tables/os_support.yml

```
@@ -190,4 +190,16 @@
     last_version: null
   status: active
   deprecation_date: null
-  eol_link: null
\ No newline at end of file
+  eol_link: null
+- name: Amazon Linux 2022
+  enterprise: true
+  enterprise_versions:
+    first_version: 3.1.x
+    last_version: null
+  oss: false
```

I might've been wrong about this now being supported for OSS. There is a kong/kong PR for adding it: https://github.com/Kong/kong/pull/9712 

Let's set it as supported and verify with Colin Hutchinson.

```suggestion
  oss: true
```